drivers/Makefile.dep

```diff
@@ -251,3 +251,18 @@ ifneq (,$(filter adc%1c,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
   USEMODULE += adcxx1c
 endif
+
+ifneq (,$(filter winc1500,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_gpio
+  FEATURES_REQUIRED += periph_spi
+  USEMODULE += xtimer
+  USEMODULE += core_mbox
+  # Import WINC1500 API from Atmel
+  USEPKG += winc1500
+  # This Wi-Fi driver can be used without NETDEV_ETH
+  ifneq (,$(filter gnrc_netdev,$(USEMODULE)))
+    USEMODULE += luid
+    USEMODULE += netif
+    USEMODULE += netdev_eth
+  endif
```

Comment:
Sorry to ask for another GNRC change :(, but `gnrc_netdev` is about to be phased out and to be replaced with `gnrc_netif2` in #7925 (hopefully the last big network related API change for a long while). Can you please port to `gnrc_netif2` directly? As far as I understand your code you only need to remove this `ifneq` block (maybe keep an `ifneq (,$(filter gnrc,$(USEMODULE); USEMODULE += luid; endif`, please test that).

Reply:
Oh will it be completely phased out? It shouldn’t be not a problem. Just wondering if `gnrc_netif2` is currently working well with enc28j60 (I am testing with this) in the master branch.

Reply:
Not on master at the moment (the integration of `gnrc_netif2` isn't merged yet), but you can try with #7479.

Reply:
#7925 is now merged to master (and `gnrc_netif2` is now called `gnrc_netif`)

Reply:
It's ported to `gnrc_netif`, but haven't tested yet.